app/helpers/sage/components_helper.rb

```diff
@@ -1,10 +1,18 @@
 module Sage
   module ComponentsHelper
+
+    # Lists out all the available sage components
     def sage_components
       [
-        { title: "Banner", path: "banner" },
-        { title: "Panel", path: "panel" },
+        { title: "banner" },
+        { title: "panel" },
       ]
     end
+    
+    # Sorts available components based on alphabet
+    def sorted_sage_components
+      sage_components.sort_by { |h| h.first }
```

Comment:
```suggestion
      sage_components.sort_by { |h| h[:title] }
```
Since the sort element is the full hash, the element ordering matters.
```ruby
> [{title: "b"},{zoo: "animal", title: "a"}].sort_by { |h| h.first }
=> [{:title=>"b"}, {:zoo=>"animal", :title=>"a"}]
```
This is optional since we're defining the hashes, but it could trip someone up if they change the structure of the hash. If this behavior is what's intended, then disregard this suggestion.